scalafix-core/shared/src/main/scala/scalafix/rule/RuleCtx.scala

```diff
@@ -51,9 +51,9 @@ trait RuleCtx extends PatchOps {
   // Private scalafix methods, subject for removal without notice.
   private[scalafix] def toks(t: Tree): Tokens
   private[scalafix] def config: ScalafixConfig
-  private[scalafix] def printLintMessage(
+  private[scalafix] def reportLintMessage(
       msg: LintMessage,
-      owner: RuleName): Unit
+      owner: RuleName): Boolean
```

Comment:
Mima may complain over this change, we can safely add a filter for it in project/Mima.scala since it's a private api.

Reply:
If it's a private api we should mark this `private[scalafix]`